index.rst

```diff
@@ -50,7 +50,696 @@
+| number               | uint64   | NOTNULL                                                   |
++----------------------+----------+-----------------------------------------------------------+
+| camera_id            | uint64   | NOT NULL REFERENCES Camera (camera_id)                    |
++----------------------+----------+-----------------------------------------------------------+
+| physical_filter_id   | uint64   | NOT NULL REFERENCES AbstractFilter (abstract_filter_id)   |
++----------------------+----------+-----------------------------------------------------------+
+| obs_begin            | datetime | NOT NULL                                                  |
++----------------------+----------+-----------------------------------------------------------+
+| obs_end              | datetime | NOT NULL                                                  |
++----------------------+----------+-----------------------------------------------------------+
+| region               | blob     |                                                           |
++----------------------+----------+-----------------------------------------------------------+
+| UNIQUE (number, camera_id)                                                                  |
++---------------------------------------------------------------------------------------------+
+
+Entries in the Visit table correspond to observations with the full camera at
+a particular pointing, possibly comprised of multiple exposures (Snaps).  A
+Visit's ``region`` field holds an approximate but inclusive representation of
+its position on the sky that can be compared to the ``regions`` of other
+DataUnits.
+
++----------------------+--------+-----------------------------------------------------------+
+| *ObservedSensor*                                                                          |
++======================+========+===========================================================+
+| observed_sensor_id   | uint64 | PRIMARY KEY                                               |
++----------------------+--------+-----------------------------------------------------------+
+| physical_sensor_id   | uint64 | NOT NULL REFERENCES PhysicalSensor (physical_sensor_id)   |
++----------------------+--------+-----------------------------------------------------------+
+| visit_id             | uint64 | NOT NULL REFERENCES Visit (visit_id)                      |
++----------------------+--------+-----------------------------------------------------------+
+| region               | blob   |                                                           |
++----------------------+--------+-----------------------------------------------------------+
+| UNIQUE (physical_sensor_id, visit_id)                                                     |
++-------------------------------------------------------------------------------------------+
+
+An ObservedSensor is simply a combination of a Visit and a PhysicalSensor, but
```

Comment:
Shouldn't it be a combination of a `Snap` and a `PhysicalSensor`?

Reply:
I suppose the nomenclature implies that a bit, but the much more important DataUnit combination is this one, as it reflects the combination of all of the Snaps in each Visit.